app/User.php

```diff
@@ -32,7 +32,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password', 'first_name', 'last_name', 'phone_number'
+        'name', 'email', 'password', 'password_confirmation','first_name', 'last_name', 'phone_number'
```

Comment:
The password confirmation field is not meant to be in the database. It is only for validation